trillian.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: trillian
+  version: 1.5.2
+  epoch: 0
+  description: Merkle tree implementation used in Sigstore
+  copyright:
+    - license: Apache-2.0
+  checks:
+    disabled:
+      - empty
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/google/trillian
+      tag: v${{package.version}}
+      expected-commit: 8ba85b59c6eca840b1b673b7afc6c9516329a690
+
+  - runs: |
+      # ensure clean git tree
+      rm .gitconfig
+      export GOCACHE=$(mktemp -d)/.cache
+      export GOMODCACHE=$(mktemp -d)/mod
+
+      go build -o trillian_log_server ./cmd/trillian_log_server
+      go build -o trillian_log_signer ./cmd/trillian_log_signer
```

Comment:
We should use `-trimpath -ldflags "-s -w"`

Reply:
thanks, done!